bwc-test/src/test/java/SecurityBackwardsCompatibilityIT.java

```diff
@@ -52,16 +66,9 @@ protected final Settings restClientSettings() {
                 .build();
     }
 
-    public void testPluginUpgradeInAMixedCluster() throws Exception {
-        assertPluginUpgrade("_nodes/" + CLUSTER_NAME + "-0/plugins");
-    }
-
-    public void testPluginUpgradeInAnUpgradedCluster() throws Exception {
```

Comment:
why remove these tests? If I understand correctly, they are testing backward compatibility during upgrade

Reply:
More details here
https://github.com/opensearch-project/security/pull/2253/files/aca655b9ffef0411e414a64611d1cf28cf31d03b#r1021991312

These tests are not testing functionality of the security plugin and to only run certain tests during certain times a bunch of logic was in the gradle script.  Until we have a richer set of tests it seems easier to start with a test and then build out how it should run based on the different circumstances